### PR TITLE
add new fluent demo app asset for ios and macos.

### PR DESCRIPTION
**Pull request by harrieshin**
### Platforms Impacted
- [x] iOS
- [x] macOS

### Description of changes

update fluent demo app icon
update xcode scheme for macOS to get rid of warnings

### Verification

Screen Shot 2022-08-05 at 4 49 37 PM

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [ ] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/1140)

### CODE REVIEW

File: macos/xcode/FluentUI.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1067,27 +1067,84 @@
...
+				CLANG_WARN_UNREACHABLE_CODE = YES;
+				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
+				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
+				GCC_WARN_ABOUT_RETURN_TYPE = YES;
+				GCC_WARN_UNDECLARED_SELECTOR = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
```
Comment (huwilkes) [inline macos/xcode/FluentUI.xcodeproj/project.pbxproj:1147]: Are these changes intentional?

Comment (harrieshin) [reply macos/xcode/FluentUI.xcodeproj/project.pbxproj:1147]: yes they are! there were bunch of xcode warnings

File: ios/FluentUI.Demo/FluentUI.Demo/Resources/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -1,107 +1,116 @@
...
-      "scale" : "3x"
+      "scale" : "3x",
+      "size" : "60x60"
     },
     {
+      "filename" : " 20.png",
       "idiom" : "ipad",
-      "size" : "20x20",
-      "scale" : "1x"
+      "scale" : "1x",
```
Comment (huwilkes) [inline ios/FluentUI.Demo/FluentUI.Demo/Resources/Assets.xcassets/AppIcon.appiconset/Contents.json:54]: Do we need the 1x images? Does anything use them?

**Review by huwilkes — APPROVED** [review]

File: ios/FluentUI.Demo/FluentUI.Demo/Resources/Assets.xcassets/AppIcon.appiconset/Contents.json (hunk 2 above)
Comment (harrieshin) [reply ios/FluentUI.Demo/FluentUI.Demo/Resources/Assets.xcassets/AppIcon.appiconset/Contents.json:54]: not sure. maybe not? but it was already in the appicon template so i just add them in any way.

**Review by Saadnajmi — APPROVED** [review]
